_gsdocs-proposals/2020/proposal_CVMFSPodmanIntegration.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Podman CVMFS integration
+layout: default
+project: CernVM-FS
+year: 2020
+organization:
+ - CERN
+---
+
+## Description
+
+[CernVM-FS (CVMFS)][cvmfs] is a globally-distributed filesystem used to lazily distribute software in different server farm.
```

Comment:
Instead of "lazily ..." I'd suggest "used to efficiently distribute software to data centers and end-user laptops"